bands/templates/bands/agreement-base.html

```diff
@@ -1,196 +1,7 @@
-            <p class="help-text">
-                To ensure that we have assigned the correct number of badges to your band, please confirm the number
-                of performers in your band.
-            </p>
-        </div>
-    </div>
-
-    <div class="form-group">
-        <label class="col-sm-3 control-label">PoC Phone Number</label>
-        <div class="col-sm-6">
-            <input class="form-control" type="text" name="poc_phone" value="{{ band_info.poc_phone|default(band.group.leader.cellphone) }}" />
-            <p class="help-text">
-                Please enter the cellphone number of your preferred Point of Contact for any issues we may need to
-                contact you about on-site.
-            </p>
-        </div>
-    </div>
-
-    <div class="form-group">
-        <label class="col-sm-3 control-label">Arrival Time</label>
-        <div class="col-sm-6">
-            <input class="form-control" type="text" name="arrival_time" value="{{ band_info.arrival_time }}" />
-            <p class="help-text">
-                Please be as specific as possible about the day and time you plan on arriving at {{ c.EVENT_NAME }}.
-            </p>
-        </div>
-    </div>
-
-    <div class="form-group">
-        <label class="col-sm-3 control-label">Parking</label>
-        <div class="checkbox col-sm-9">
-            {{ macros.checkbox(band_info, 'bringing_vehicle', label='We will be parking at ' + c.EVENT_NAME) }}
-        </div>
-    </div>
-
-    <div id="vehicle-info" class="form-group" style="display:none">
-        <label class="col-sm-3 control-label">Vehicle Information</label>
-        <div class="col-sm-6">
-            <input class="form-control" type="text" name="vehicle_info" value="{{ band_info.vehicle_info }}" />
-            <p class="help-text">
-                If you're parking on-site, then please tell us the vehicle info (car, truck, attached trailer, tour bus, monster truck), so we can ensure parking availability.
-            </p>
-        </div>
-    </div>
-
-    <h3>
-        Performance Agreement
-    </h3>
-    <p>
-        By clicking below and submitting your information, you agree to the terms of this performance agreement.
-    </p>
-
-    <div class="form-group">
-        <div class="col-sm-6 col-sm-offset-3">
-            <button type="submit" class="btn btn-primary">
-                Save Information and Agree
-            </button>
-        </div>
-    </div>
-</form>
-
-<script>
-    var showOrHideVehicleInfo = function () {
-        setVisible('#vehicle-info', $.field('bringing_vehicle').prop('checked'));
-    };
-    $(function () {
-        showOrHideVehicleInfo();
-        $.field('bringing_vehicle').on('click', showOrHideVehicleInfo);
-    });
-</script>
-
-{% endblock band_information_form %}
+  {% include "bands/" + group_type + "_agreement.html" %}
```

Comment:
Much like the admin info section, the agreement pages for different group types are assumed to need to be unique per group type.